### ISSUE TITLE
Chatbot now only creates Python code

### ISSUE THREAD

**Issue by angelhodar** [vercel/ai-chatbot#654]
I have updated the code and now the chatbot refuses to create any code that is not Python. You can test it in chat.vercel.ai as well

![image](https://github.com/user-attachments/assets/010167ed-5fd2-4048-a998-5d6bc2902fb0)

I know its because of this line in the first prompt in ``prompts.ts``

```
When asked to write code, always use blocks. When writing code, specify the language in the backticks, e.g. \`\`\`python\`code here\`\`\`. The default language is Python. Other languages are not yet supported, so let the user know if they request a different language.
```

But it should be able to just generate plain code in other languages even if its not executable. I have modified the prompt but the generated typescript code is not shown formatted, only python code is formatted somehow.

**Comment by JuniYadi** [comment]
<img width="1155" alt="Screenshot 2024-12-23 at 17 32 47" src="https://github.com/user-attachments/assets/a3fc8e85-04f9-4f95-8801-c6d8917056a6" />


we can make a different language but it's still a prompt console for Python, probably needs changes too

change the codePrompt => https://github.com/vercel/ai-chatbot/blob/main/lib/ai/prompts.ts

**Comment by athrael-soju** [comment]
This is intentional, as the app uses pyodite to run the generated Python code.

**Comment by adiguno** [comment]
> This looks intentional, as the app uses pyodide to run the generated Python code.

yeah, say it in `app/(chat)/layout.tsx`. What's the point of this?
```tsx
      <Script
        src="https://cdn.jsdelivr.net/pyodide/v0.23.4/full/pyodide.js"
        strategy="beforeInteractive"
      />
```

**Comment by aryadhruv** [comment]
I update the prompt.ts in lib/ai and it worked without any issue. Sure, console doesn't support anything other than Python, but there's not point in restrcting users from not able to generate response in language they want
